packages/gatsby/src/bootstrap/load-plugins/validate.ts

```diff
@@ -197,16 +197,23 @@ export async function validatePluginOptions({
           }
 
           try {
-            // All plugins have "plugins: []"" added to their options in load.ts, even if they
-            // do not have subplugins. We add plugins to the schema if it does not exist already
-            // to make sure they pass validation.
+            if (typeof plugin.pluginOptions === `undefined`) {
```

Comment:
for some reason `pluginOptions` can be undefined, so I've handled that case (thanks ts).

Reply:
Do you have a repro cause it shouldn't

Reply:
Not specifically. But type script believed it could be. I'm guessing TS is wrong but I didn't try and hunt down where that was coming from. I can try and figure that out and let you know.

Reply:
that means our gatsby types are wrong I guess ^^

Reply:
@mxstbr this can be done in a follow-up PR but I would like to see where this comes from cause it means something in our types are wrong

Reply:
https://github.com/gatsbyjs/gatsby/blob/10d345111d37f6c4ef1cfd896d050f8dfdd2d349/packages/gatsby/src/bootstrap/load-plugins/types.ts#L22-L24